frontend/components/TooltipWrapper/TooltipWrapper.tsx

```diff
@@ -71,7 +77,8 @@ const TooltipWrapper = ({
           id={tipId}
           delayShow={isDelayed ? 500 : undefined}
           delayHide={isDelayed ? 500 : undefined}
-          noArrow
+          // noArrow={!showArrow}
+          noArrow={false}
```

Comment:
is this meant to be hardcoded as false?

Reply:
oppse sorry, this is wrong, I will change this.

Reply:
its not meant to be hardcoded

Reply:
@RachelElysia ok i fixed this.